terraform/layer2-k8s/eks-elk.tf

```diff
@@ -18,7 +18,7 @@ locals {
   elk_elasticsearch_values    = <<VALUES
 elasticsearch:
   enabled: true
-  image: "halfb00t/elasticsearch"
+  image: "public.ecr.aws/o7m5y2d9/elasticsearch"
```

Comment:
is it an official elasticsearch docker image?
does it contain s3-plugin?

Reply:
Yes it is, it's built using this [Dockerfile](https://github.com/maddevsio/aws-eks-base/blob/main/docker/elasticsearch/Dockerfile)